packages/xdg_directories/example/lib/main.dart

```diff
@@ -0,0 +1,140 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:xdg_directories/xdg_directories.dart';
+import 'widgets/directory_display.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'XDG Directories Demo',
+      theme: ThemeData(
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
```

Comment:
Why is this using a custom color scheme? The example should be focused on the package itself.

Reply:
I believe it came by default when creating a new Flutter project on Ubuntu. Removing it now...